ussologin/ussologin.go

```diff
@@ -69,13 +73,16 @@ var loginForm = form.Form{
 		otpKey: environschema.Attr{
 			Description: "Two-factor auth",
 			Type:        environschema.Tstring,
+			Mandatory:   true,
```

Comment:
Not sure what the significance of mandatory is. It's possible for users to not have 2 factor turned on in which case empty is valid here